bin/publish.ls

```diff
@@ -0,0 +1,138 @@
+require! {
+  'chalk' : {bold, red, cyan, yellow, green}
+  'inquirer'
+  'jsonfile'
+  'semver'
+  'shelljs/global'
+}
+
+{name, version} = jsonfile.readFileSync './exocom-server/package.json' # not sure about this, using exocom-server b/c there is no root package.json
```

Comment:
Sounds good to me. Please change the comment to explain this without asking a question.